adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Microsoft.IdentityModel.Clients.ActiveDirectory.csproj

```diff
@@ -148,4 +162,21 @@
   <ItemGroup>
     <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="All" />
   </ItemGroup>
+
+  <Target Name="LayoutNuGet">
+    <PropertyGroup>
+      <NugetPackageRootFolder Condition="'$(NugetPackageRootFolder)' == ''">$([System.IO.Path]::Combine($(MSBuildThisFileDirectory), "..",".nugetlayout"))</NugetPackageRootFolder>
+      <NuspecFilePath>$([System.IO.Path]::Combine($(MSBuildThisFileDirectory), "..", "..", "build", "Microsoft.Identity.Client.nuspec"))</NuspecFilePath>
+    </PropertyGroup>
+    <ItemGroup>
+      <FilesToPack Include="bin\$(Configuration)\**\Microsoft.IdentityModel.Clients.ActiveDirectory.dll" />
+      <FilesToPack Include="bin\$(Configuration)\**\Microsoft.IdentityModel.Clients.ActiveDirectory.xml" />
+      <FilesToPack Include="bin\$(Configuration)\**\Microsoft.IdentityModel.Clients.ActiveDirectory.pdb" />
```

Comment:
><FilesToPack Include="bin\$(Configuration)\**\Microsoft.IdentityModel.Clients.ActiveDirectory.pdb" /> [](start = 6, length = 101)

This is only added in debug, right?

Reply:
No, you can have PDB files in Release as well, although I am not sure about the debugging experience. In any case, the packaging task will create 2 packges - one with the PDBs and one without. NuGet should add support for symbol packages at some point.

Reply:
Yeah, however do we generate it today during build? I don't think we do? We should have symbols for release bits though. Those should be on the public symbols server though (which I believe they are).

Reply:
We do generate PDB files through build and we publish them to internal Microsoft server. Still trying to figure out how to publish them to a public server, perhaps @brentschmaltz knows more?